pyartcd/pyartcd/pipelines/release_from_fbc.py

```
@@ -941,6 +945,39 @@ async def _set_shipment_mr_dependency(self, blocking_mr_url: str):
+            self.logger.warning("Could not extract JIRA issue key from URL: %s", self.release_jira)
+            return
+
+        if self.dry_run:
+            self.logger.info("[DRY-RUN] Would add remote link on %s pointing to %s", issue_key, mr_url)
+            return
+
+        try:
+            jira_client = self.runtime.new_jira_client()
+            jira_client.add_remote_link(issue_key, {"title": "Shipment MR", "url": mr_url})
+            self.logger.info("Added shipment MR link to JIRA ticket %s", issue_key)
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)